superset/Dockerfile

```diff
@@ -2,8 +2,21 @@ FROM apache/superset:2.0.0
 # Switching to root to install the required packages
 USER root
 
+ARG SUPERSET_PASSWORD
+ENV SUPERSET_PASSWORD $SUPERSET_PASSWORD
+
 RUN pip install psycopg2
 
 RUN pip install psycopg2-binary
```

Comment:
I do not think we need to install the two psycopg2 variants. The former would be used when you need to build it from source while the latter is used when you want a pre-built package https://pypi.org/project/psycopg2-binary/

psycopg2-binary would be appropriate for this scenario.